src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -1119,7 +1143,6 @@
     <GenericsSources Include="$(BclSourcesRoot)\System\Collections\Concurrent\PartitionerStatic.cs" />
   </ItemGroup>
   <ItemGroup>
-    <SafehandleSources Include="$(BclSourcesRoot)\Microsoft\Win32\SafeHandles\SafeFileHandle.cs" />
```

Comment:
@jkotas I'm replacing existing uses of SafeFileHandle (which are all internal) with another SafeHandle that is explicit about using Win32Interop to close. Seems relatively constrained- I'll have an update soon.